apps/judicial-system/backend/src/app/modules/police/police.service.ts

```diff
@@ -34,7 +34,6 @@ import { AwsS3Service } from '../aws-s3'
 import { Case } from '../case'
 import { Defendant } from '../defendant/models/defendant.model'
 import { EventService } from '../event'
-import { Subpoena, SubpoenaService } from '../subpoena'
```

Comment:
I had some trouble with circular dependencies so I removed these imports. The subpoena service was not being used and only the id from subpoena was being accessed.